polar-core/tests/integration_tests.rs

```diff
@@ -1675,6 +1675,83 @@ fn test_circular_data() -> TestResult {
     Ok(())
 }
 
+fn commute_ops(u: &Value, v: &Value) -> bool {
```

Comment:
Pls add a comment explaining what this does / how it works for us mere mortals. I'm having trouble following the control flow